src/components/SwapWidget/index.tsx

```diff
@@ -8,14 +8,29 @@ export interface Props {
   isLimitOrderVisible?: boolean;
   showSettings?: boolean;
   partnerDaaS?: string;
+  defaultInputToken?: string;
+  defaultOutputToken?: string;
 }
 
 const SwapWidget: React.FC<Props> = ({
   isLimitOrderVisible = false,
   showSettings = true,
   partnerDaaS = ZERO_ADDRESS,
+  defaultInputToken,
+  defaultOutputToken,
 }) => {
   const [swapType, setSwapType] = useState('MARKET' as string);
+  const [defaultInputCurrency, setDefaultInputCurrency] = useState(defaultInputToken as string);
+  const [defaultOutputCurrency, setDefaultOutputCurrency] = useState(defaultOutputToken as string);
+
+  const updateDefaultInputCurrency = (value: string) => {
+    setDefaultInputCurrency(value);
+  };
+
+  const updateDefaultOutputCurrency = (value: string) => {
+    setDefaultOutputCurrency(value);
+  };
```

Comment:
we don't need these functions. if you need you can directly pass `setDefaultOutputCurrency` & `setDefaultInputCurrency` to props like

`updateDefaultInputCurrency={setDefaultInputCurrency}`
`updateDefaultOutputCurrency={setDefaultOutputCurrency}`